feel-engine/src/main/scala/org/camunda/feel/FeelEngine.scala

```diff
@@ -152,8 +167,19 @@ class FeelEngine(val functionProvider: FunctionProvider =
         Left(Failure(s"failed to parse expression '$expression': $e"))
     }
 
+  private def validate(
+      exp: ParsedExpression): Either[Failure, ParsedExpression] = {
+
+    validator
+      .validateExpression(exp.expression)
+      .map(failure =>
+        Failure(
+          s"""validation of expression '${exp.text}' failed: ${failure.message}"""))
+      .toLeft(exp)
+  }
+
   def eval(exp: ParsedExpression, context: Context): EvalExpressionResult =
```

Comment:
Question (based in insufficient Scala knowledge): Are these functions final, or is there a way to override these functions and skip the validation?

Reply:
As a user of the FEEL engine, there is no way to skip the validation.